### PR TITLE
NOBUG: Update deployment config to handle new variables, and deploy p…

### PR DESCRIPTION
**Pull request by marklise**
…roperly.

### CODE REVIEW

File: openshift/parse-deployment-template.json [hunk 1]
```diff
@@ -427,29 +546,95 @@
...
       "description": "A unique identifier for your app",
-      "value": ""
+      "value": "changeMe"
     },
     {
       "name": "MASTER_KEY",
       "displayName": "Parse Master Key",
       "description": "A key that overrides all permissions. Keep this secret",
-      "value": ""
+      "value": "changeMe"
```
Comment (ActionAnalytics) [inline openshift/parse-deployment-template.json:559]: Should this be a variable or is this OK?

**Review by ActionAnalytics — CHANGES_REQUESTED** [review]
Let's finish the variables that are tagged 'changeMe'

File: openshift/parse-deployment-template.json (hunk 1 above)
Comment (marklise) [reply openshift/parse-deployment-template.json:559]: This is representing the default value of 'changeMe' when the user imports this into openshift and clicks "process template".  They'll see this as one of the fields and hopefully get the hint to change the value.  There are committed defaults in index.js, so maybe we should remove this and fix up the index.js to fail startup if the env var is not set?

Comment (ActionAnalytics) [reply openshift/parse-deployment-template.json:559]: That explanation makes sense to me, thank you.

**Pull request merged**